src/components/new-conversation-dialog/create-room/index.js

```diff
@@ -39,26 +40,31 @@ const CreateRoom = ({
   onCreateRoom,
 }) => {
   useEffect(() => {
-    onSearchMembers()
+    // onSearchMembers()
   }, [])
 
   const isRowLoaded = useCallback(({ index }) => Boolean(members[index]), [
     members,
   ])
 
+  const [hoverBack, setHoverBack] = useState(false)
+
   return (
     <Flex direction="column" items="start" className={classes.wrapper}>
-      <ActionLink
-        className={classes.link}
-        variant="basic"
-        onClick={hideCreateRoom}
-        href="#"
-        icon="arrowLeft"
-      >
-        Create a new group
-      </ActionLink>
+      <div className={classes.backWrapper}>
+        <button
```

Comment:
We have a `Button` component with icon only. Or should another hover color be applied?

Should be:
```
<Button
    className={classes.back}
    onClick={hideCreateRoom}
    variant="basic"
    appearance="minimal"
    size="regular"
    icon="arrowLeft"
  />
  ```
  
  in case we don't need another color on hover

Reply:
Just discussed this with Carina. This should be a link with icon inside. Something like this:

```
<a href="#groups" className={classes.back}>
  <Icon name="arrowLeft" />
</a>
```

We need to create a Link component for such cases in future

Reply:
ok, but then we also need to handle the `href="#groups"` . I used a button because I wanted a click handler but `<a href...>` would be better.

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md